.editorconfig

```diff
@@ -63,6 +63,9 @@ dotnet_style_readonly_field = true:suggestion
 # Parameter preferences
 dotnet_code_quality_unused_parameters = all:suggestion
 
+# Diagnostic preferences
+dotnet_diagnostic.CA5394.severity = none # Do not use insecure randomness
```

Comment:
Otherwise, can't use the pseudo-random generator System.Random.

Reply:
Instead we should be using ` RandomNumberGenerator.GetInt32` like we do in IceRPC.

Reply:
Will fix in a follow-up PR.